src/status_im2/utils/simple_message_resolver.cljs

```diff
@@ -0,0 +1,24 @@
+(ns status-im2.utils.simple-message-resolver
```

Comment:
nit: why not `status-im2.utils.message-resolver` -> do we have other variants of message resolvers?

Reply:
Related to this, even if we had other variants, in Clojure I think it's more common to avoid namespace proliferation, since it's not like we need one class for each file, or in other words, one resolver for each file. Probably as you suggested @J-Son89 is enough to be future proof in case other resolvers need to be defined.

Reply:
We have some, at least two other cases. The naming is kind of arbitrary here, I can generalise it to `message-resolver` but I don't want to work on moving other cases of it to this ns (at least not under this PR).